DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1

```diff
@@ -434,7 +494,7 @@ function Get-ChocoInstalledPackage {
     $ChocoInstallList = Join-Path -Path $ChocoInstallLP -ChildPath 'ChocoInstalled.xml'
 
     if ($Purge.IsPresent) {
-        Remove-Item $ChocoInstallList -Force
+        Remove-Item $ChocoInstallList -Force -ErrorAction SilentlyContinue

```

Comment:
I think we should know if the purge fails so I'd rather see some code checking if it the files exists and then removing it? If there are errors I think we should throw the exception.